transmart-rest-api/src/main/groovy/org/transmartproject/rest/dataExport/ExportJobExecutor.groovy

```diff
@@ -44,10 +45,10 @@ class ExportJobExecutor implements InterruptableJob, AutoCloseable {
         Long jobId = jobDataMap.jobId
         String jobName = jobDataMap.jobName
         User user = jobDataMap.user
-        String file = getFilePath(user, "${jobName}.zip")
+        String file = getFilePath(user, jobName)
```

Comment:
Does it mean the following requirement taken from the ticket:
`Change the naming convention for exports to include the time it was created (or last updated) e.g.: export-for-wim_2018-04-15_1246.zip`